ros/api/v1/hosts.py

```diff
@@ -27,28 +29,34 @@
 
 class IsROSConfiguredApi(Resource):
     def get(self):
-        ident = identity(request)['identity']
-        account_query = db.session.query(RhAccount.id).filter(
-            RhAccount.account == ident['account_number']).subquery()
-        system_count = db.session.query(System.id)\
-            .filter(System.account_id.in_(account_query)).count()
-        systems_with_suggestions = db.session.query(System.id)\
-            .filter(System.account_id.in_(account_query))\
-            .filter(System.number_of_recommendations > 0).count()
-        systems_waiting_for_data = db.session.query(System.id)\
-            .filter(System.account_id.in_(account_query))\
-            .filter(System.state == 'Waiting for data').count()
+        account_number = identity(request)['identity']['account_number']
+        system_query = default_queries(account_number)
 
-        if system_count <= 0:
-            return {
-                'success': False,
-                'code': 'NOSYSTEMS',
-                'count': system_count
-            }
+        last_reported = (
```

Comment:
I would like to understand why did you add this extra query?

Reply:
@kgaikwad IMO, inside the /systems api, systems are listed based on Performance Profile filter i.e. we show systems only which have Performance Profile associated with the respective system.
It would make more sense to use the same query inside /is_configured api to keep the system counts consistent in both api responses.
Thoughts?